modules/cp4aiops/scripts/aimanager/update_cert.sh

```diff
@@ -1,13 +1,17 @@
 #!/bin/sh
 
-# NAMESPACE="cp4aiops"
-
 ########################
 #
 # A signed certificate is needed on the NGNIX pods for the Slack and Teams integrations.
 #
 #######################
 
+# Check for external-tls-secret before the start of updating cert
+if [ "`kubectl get secret -n $NAMESPACE external-tls-secret --ignore-not-found`" == "" ]; then
+  # Creates external-tls-secret if it's missing
+  oc get secret internal-nginx-svc-tls --namespace=${NAMESPACE}  -o yaml | sed -e "s/internal-nginx-svc-tls/external-tls-secret/g" | kubectl apply -n ${NAMESPACE} -f -
```

Comment:
Just curious why you used `kubectl` above and `oc` here.  Can `oc` be replaced by `kubectl`?

Reply:
It has been replaced with `kubectl`. I missed it in the original pass.